src/images/icons/share.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```

Comment:
I think this should go in _includes/icons so we can inline it with the `icon()` macro